Makefile

```diff
@@ -111,8 +111,8 @@ venv_osx:
 clean_venv:
 	rm -rf venv
 
-PIP_DEV_DEPS_CMD = $(MAKE) -C $(CC_ANALYZER) pip_dev_deps && \
-  $(MAKE) -C $(CC_WEB) pip_dev_deps && \
+PIP_DEV_DEPS_CMD = $(MAKE) -C $(CC_ANALYZER) -j1 pip_dev_deps && \
+  $(MAKE) -C $(CC_WEB) -j1 pip_dev_deps && \
```

Comment:
Why do we need `j1` here? The `pip_dev_deps` target in both cases will call a single pip install command:

https://github.com/Ericsson/codechecker/blob/f11eb1a09c796a083a8fb9ea7d94946a262fe8da/analyzer/Makefile#L29-L30
https://github.com/Ericsson/codechecker/blob/f11eb1a09c796a083a8fb9ea7d94946a262fe8da/web/Makefile#L50-L51

Reply:
I think I tried this, but let me try again.